packages/extension-long-memory/src/layers/emgas/extractor.ts

```
@@ -54,8 +58,11 @@ export async function extractGraphElements(
     text: string
 ): Promise<ExtractedGraphElements | null> {
     if (!modelRef.value) {
-        logger.warn(
-            'LLM-based extractor is disabled. Cannot extract graph elements.'
+        throw new ChatLunaError(
+            ChatLunaErrorCode.MODEL_NOT_FOUND,
+            new Error(
+                'LLM-based extractor is disabled. Cannot extract graph elements.'
+            )
         )
         return null
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)